Refine configuration and content sections. (#16)
canopy-iiif/docs: pages/configuration/search.mdx

| `flexSearch` | `Object`  | FlexSearch configuration options. |
| `index`      | `Object`  | Indexing option for Manifests.    |

### enabled

Boolean value to enable or disable search indexing. Default is `true`.

```json
"enabled": true
```

### flexSearch

Users can customize their [FlexSearch configuration](https://github.com/nextapps-de/flexsearch#options) using `search.flexSearch` to fit around the source Collection and its Manifest `label`, `metadata`, and `summary` properties. Customizations range from defining language-specific options such as `charset` and `stemmer`, as well as scoring options like `resolution` and `depth`.

```json
"flexSearch": {
    "charset": "latin:extra",
    "optimize": true,
    "tokenize": "strict",
    "bidirectional": false,
    "document": {
      "index": [
        {
          "field": "label",
          "tokenize": "full",
          "resolution": 9,
          "depth": 3,
          "bidirectional": true
        },
        {
          "field": "metadata",
          "resolution": 2
        },
        {
          "field": "summary",
          "resolution": 1
        }
      ]
    }
  }
```

### index

> Option available for implementers to define what values of Manifest are indexed. Canopy IIIF indexes the `label` property on the Manifest by default. Users can also index the `metadata` and `summary` properties.

Reviewer: ```suggestion
Implementers can define the properties of their manifests that are indexed. Canopy IIIF indexes the `label` property on the Manifest by default. Users can also index the `metadata` and `summary` properties.
```
